dummy_robot/dummy_robot_bringup/launch/dummy_robot_bringup.launch.py

```diff
@@ -17,19 +17,36 @@
 from launch import LaunchDescription
 from launch_ros.actions import Node
 from launch_ros.substitutions import FindPackageShare
+from launch.actions import DeclareLaunchArgument
```

Comment:
The order of these import statements should be the opposite : https://ci.ros2.org/job/ci_linux/18157/testReport/junit/dummy_robot_bringup/flake8/I100____launch_dummy_robot_bringup_launch_py_20_1_/. 

Looks good to me otherwise, and it seems there are no dependency cycles. I'm willing to approve this once the flake error is resolved. What do you think @audrow ?

Reply:
Placed `from launch.actions import DeclareLaunchArgument` on top of `from launch_ros.substitutions import FindPackageShare`.